packages/documentation/copy/en/handbook-v2/Classes.md

```diff
@@ -150,11 +150,24 @@ class Point {
 
 ```ts twoslash
 class Point {
-  // Overloads

```

Comment:
I think the point of this section is mainly just to explain the syntax; probably the body could just be stubbed out and you can keep the new overloads (which are better)

Reply:
(Marking as needing an update)